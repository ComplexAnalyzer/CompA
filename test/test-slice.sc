def main(){
   a = [1,2,3,4,5,6,7,8,9,10];
   b = a;
   c = b;
   d = a[1:4];
   (0<=i<#d) print(d[i]); 
   e = [1.1, 3.3, 4.4, 5.5, 6.6];
   f = e[0:2];
   (0<=i<#f) print(f[i]); 
   g = ["hi", "see", "ya", "later"];
   h = g[1:#g];
   (0<=i<#h) print(h[i]); 

}

