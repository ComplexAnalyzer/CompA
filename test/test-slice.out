2
3
4
1.100000
3.300000
see
ya
later
